noxfile.py

```diff
@@ -172,6 +172,10 @@ def lint(session: nox.Session) -> None:
     session.run("pre-commit", "run", *args)
 
 
+# NOTE: This session will COMMIT upgardes to vendored libraries.
+# You should therefore not run it directly against main, as you
+# will put your main branch out of sync with upstream. Always run
+# it on a dedicated branch
```

Comment:
Typo and... let's include instructions for how to fix things instead of how to change things.

```suggestion
# NOTE: This session will COMMIT upgrades to vendored libraries.
# You should therefore not run it directly against `main`. If you
# do (asusming you started with a clean main), you can run:
#
# git checkout -b vendoring-updates
# git checkout main
# git reset --hard origin/main
```